app/views/govuk_publishing_components/components/_govspeak.html.erb

```diff
@@ -5,6 +5,7 @@
   local_assigns[:margin_bottom] ||= 0
   direction_class = "gem-c-govspeak--direction-#{direction}" if local_assigns.include?(:direction)
   disable_youtube_expansions = local_assigns.fetch(:disable_youtube_expansions) if local_assigns.include?(:disable_youtube_expansions)
+  font_size_class = local_assigns.fetch(:font_size, "m").then { |font_size| "gem-c-govspeak--font-size-#{font_size}" }
```

Comment:
Could this be simplified we modified line 17 to something like

```Ruby
component_helper.add_class("gem-c-govspeak--font-size-l") if use_large_font
```
?

Reply:
While we only support `"l"` as an option, yes, but if we decided to support `"s"` or `"xl"` in the future, this wouldn't need updating.

Reply:
Addressed in [b38cc23](https://github.com/alphagov/govuk_publishing_components/pull/4461/commits/b38cc23c274b22c7300f0d0b7f95ae92fdaddf23)